### DISCUSSION TITLE
мусор

### DISCUSSION THREAD

**Discussion by riverzaw** [r3nya/madrid101#17] [Ideas]
пункты сбора мусора

**Comment by r3nya** [comment]
- https://www.comunidad.madrid/servicios/consumo/reciclaje-donde-tirar-cada-producto
- как искать "punto limpio" / "punto limpio mobil"

**Comment by r3nya** [comment]
- Нашел модерируемый список от администрации “puntos limpios” – https://www.google.com/maps/d/u/0/viewer?mid=15-bggcDRMhoFyxtOq2huMNe8s9CQ3-KR&ll=40.38356047845373%2C-3.6244186110595766&z=12
- https://geoportal.madrid.es/IDEAM_WBGEOPORTAL/visor_ide.iam?ArcGIS=https://sigma.madrid.es/hosted/rest/services/RESIDUOS/PUNTOS_LIMPIOS/MapServer

**Comment by r3nya** [comment]
Get pics from:
- https://www.madrid.es/UnidadesDescentralizadas/LimpiezaUrbanaYResiduos/Campa%C3%B1as/00.%20CAMPA%C3%91AS%20DE%20INFORMACI%C3%93N%20Y%20SENSIBILIZACI%C3%93N/01%20CAMPA%C3%91A%20%22Con%20Erre%20De%22/residuo-cero_folleto-gral_ayto-madrid.pdf
- https://www.madrid.es/portales/munimadrid/es/Inicio/Medio-ambiente/Recogida-de-residuos/Campana-de-separacion-de-residuos-y-limpieza-Con-erre-de/?vgnextfmt=default&vgnextoid=1be8db4c105ef710VgnVCM2000001f4a900aRCRD&vgnextchannel=f81379ed268fe410VgnVCM1000000b205a0aRCRD&idCapitulo=11713839&rm=dfa8e135da00e710VgnVCM1000001d4a900aRCRD
- https://support.urbancampus.com/recycling-in-spain

**Comment by r3nya** [comment]
https://www.madrid.es/portales/munimadrid/es/Inicio/Medio-ambiente/Residuos-y-limpieza-urbana/Puntos-Limpios-del-Ayuntamiento-de-Madrid?vgnextfmt=default&vgnextoid=a30815e08aa62110VgnVCM1000000b205a0aRCRD&vgnextchannel=f81379ed268fe410VgnVCM1000000b205a0aRCRD
